### ISSUE TITLE
Code not working for RGB images only Grayscale

### ISSUE THREAD

**Issue by Scienceseb** [sangmin-git/LMC-Memory#11]
Hello, first thanks a lot for your amazing paper !

The code right now is not working for 3 channel images even if the argument args.img_channel is put to 3.
So to make it effective with RGB images the following modifications are require:

model.py
line 15:
` nn.Conv2d(in_channels=args.img_channel, out_channels=64, kernel_size=(3, 3), stride=2, padding=1),`
line 30
` nn.ConvTranspose2d(in_channels=64, out_channels=args.img_channel, kernel_size=(3, 3), stride=2, padding=1, output_padding=1))`

and 
line 100 (this one dont work with args.img_channel, you can fix that pretty easily)
` nn.Conv3d(3, 64, kernel_size=(3, 3, 3), padding=(1, 1, 1)),`

Thanks!

**Comment by sangmin-git** [comment]
You're right. Thanks for the pointing out the issue!
